packages/admin/src/targetGroups/TargetGroupForm.tsx

```diff
@@ -27,41 +28,54 @@ import { createTargetGroupMutation, targetGroupFormQuery, updateTargetGroupMutat
 import {
     GQLCreateTargetGroupMutation,
     GQLCreateTargetGroupMutationVariables,
-    GQLTargetGroupFormFragment,
     GQLTargetGroupFormQuery,
     GQLTargetGroupFormQueryVariables,
     GQLUpdateTargetGroupMutation,
     GQLUpdateTargetGroupMutationVariables,
 } from "./TargetGroupForm.gql.generated";
 
-type FormValues = GQLTargetGroupFormFragment;
+export interface EditTargetGroupFinalFormValues {
+    title: string;
+    [key: string]: unknown;
+}
 
 interface FormProps {
     id?: string;
     scope: ContentScopeInterface;
+    additionalFormFields?: React.ReactNode;
+    nodeFragment?: { name: string; fragment: DocumentNode };
+    dataToInitialValues?: (values?: EditTargetGroupFinalFormValues) => EditTargetGroupFinalFormValues;
```

Comment:
I think I would call it input2State, would be more "comet"-like and blocks also always have the naming: input2State, output2State, state2output

or here for example: https://github.com/vivid-planet/comet/blob/main/packages/admin/cms-admin/src/common/useEditState.tsx

Reply:
I renamed the prop to `input2State` in [c23831d](https://github.com/vivid-planet/comet-brevo-module/pull/30/commits/c23831d30073b719fdca309fb65cc8b5c48c030e)